besu/src/test/java/org/hyperledger/besu/cli/BesuCommandTest.java

```diff
@@ -807,50 +807,6 @@ public void configOptionDisabledUnderDocker() {
     assertThat(commandOutput.toString()).isEmpty();
   }
 
```

Comment:
Removing these tests is correct - but we probably need to put them somewhere else - for example, there's a test which states that --node-private-key-file _cannot_ be set if we are dockerised... not sure how we're going to do that with the plugin approach :/

Reply:
They will be moving to plugin unit tests.

Reply:
created unit tests in plugin to cover these scenarios.